cluster/src/main/java/io/atomix/cluster/impl/DefaultClusterService.java

```diff
@@ -183,7 +183,7 @@ private void sendHeartbeats() {
       sendHeartbeat(node.address(), payload);
       PhiAccrualFailureDetector failureDetector = failureDetectors.computeIfAbsent(node.id(), n -> new PhiAccrualFailureDetector());
       double phi = failureDetector.phi();
-      if (phi >= phiFailureThreshold || System.currentTimeMillis() - failureDetector.lastUpdated() > DEFAULT_FAILURE_TIME) {
+      if (phi > 0 && (phi >= phiFailureThreshold || System.currentTimeMillis() - failureDetector.lastUpdated() > DEFAULT_FAILURE_TIME)) {
```

Comment:
@kuujo wouldn't 100 make sense here to keep it consistent with the Netty failure detection minimum value?

Reply:
oops no, phi, not timeout ms.